docker-compose.yml

```diff
@@ -56,52 +64,57 @@ services:
         C_FORCE_ROOT: 'true'
         ALEPH_ELASTICSEARCH_URI: http://elasticsearch:9200/
         ALEPH_DATABASE_URI: postgresql://aleph:aleph@postgres/aleph
-        POLYGLOT_DATA_PATH: /polyglot
-        TESSDATA_PREFIX: /usr/share/tesseract-ocr
```

Comment:
This was moved into the aleph_base image since that is the place where the OCR was configured and installed.